sdk/search/Microsoft.Azure.Search.Service/src/Generated/Models/ServiceLimits.cs

```diff
@@ -37,11 +37,15 @@ public ServiceLimits()
         /// <param name="maxComplexCollectionFieldsPerIndex">The maximum number
         /// of fields of type Collection(Edm.ComplexType) allowed in an
         /// index.</param>
-        public ServiceLimits(int? maxFieldsPerIndex = default(int?), int? maxFieldNestingDepthPerIndex = default(int?), int? maxComplexCollectionFieldsPerIndex = default(int?))
+        /// <param name="maxComplexObjectsInCollectionsPerDocument">The maximum
+        /// number of objects in complex collections allowed per
+        /// document.</param>
+        public ServiceLimits(int? maxFieldsPerIndex = default(int?), int? maxFieldNestingDepthPerIndex = default(int?), int? maxComplexCollectionFieldsPerIndex = default(int?), int? maxComplexObjectsInCollectionsPerDocument = default(int?))
```

Comment:
This is a binary breaking change did you guys intend to break?

Reply:
Please see the other comment - we've accepted this as a breaking change!